### PR TITLE
SCMOD-13334: Pg driver fix Update to release notes

### LINKED ISSUE #41: Tomcat-Juli Jar Causes java.lang.NoClassDefFoundError: Could not initialize class org.postgresql.Driver
If I place `tomcat/lib/postgresql-42.2.19-.jar` (or 42.2.6) and add a resource within `tomcat/conf/server.xml` that defines the connection to a PostgreSQL database, specifically with "initialSize" set, as follows:
```
    <GlobalNamingResources>
        <Resource auth="Container"
                  driverClassName="org.postgresql.Driver"
                  factory="org.apache.tomcat.dbcp.dbcp2.BasicDataSourceFactory"
                  initialSize="10"
                  name="shared/jdbc/OpsDS"
                  password="password"
                  type="javax.sql.DataSource"
                  url="jdbc:postgresql://myserver:5432/mydatabase"
                  username="mydbuser" />
    </GlobalNamingResources>
```
... and start Tomcat with these commands
``` 
      cd $CATALINA_HOME
      exec /usr/local/bin/gosu ${USER_NON_ROOT}:${USER_NON_ROOT} catalina.sh run
```
... then the CAFapi provided `tomcat/bin/tomcat-juli.jar` interferes with the Tomcat startup, resulting in this error:
```
[02:05:37.109Z #879.001 ERROR -            -   ] o.a.catalina.startup.Catalina: {
"message":"The required Server component failed to start so Tomcat is unable to start.",
"exception":"java.lang.NoClassDefFoundError: Could not initialize class org.postgresql.Driver
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.apache.tomcat.dbcp.dbcp2.DriverFactory.createDriver(DriverFactory.java:44)
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.createConnectionFactory(BasicDataSource.java:462)
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.createDataSource(BasicDataSource.java:528)
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSource.getLogWriter(BasicDataSource.java:1048)
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSourceFactory.createDataSource(BasicDataSourceFactory.java:568)
	at org.apache.tomcat.dbcp.dbcp2.BasicDataSourceFactory.getObjectInstance(BasicDataSourceFactory.java:240)
	at com.netiq.tomcat.jdbc.pool.CustomBasicDataSourceFactory.getObjectInstance(CustomBasicDataSourceFactory.java:78)
	at org.apache.naming.factory.FactoryBase.getObjectInstance(FactoryBase.java:96)
	at javax.naming.spi.NamingManager.getObjectInstance(NamingManager.java:321)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:857)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:160)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:843)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:160)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:843)
	at org.apache.naming.NamingContext.lookup(NamingContext.java:174)
	at org.apache.catalina.core.NamingContextListener.addResource(NamingContextListener.java:1017)
	at org.apache.catalina.core.NamingContextListener.createNamingContext(NamingContextListener.java:557)
	at org.apache.catalina.core.NamingContextListener.lifecycleEvent(NamingContextListener.java:253)
	at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:123)
	at org.apache.catalina.core.StandardServer.startInternal(StandardServer.java:922)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	... 7 common frames omitted
```
Using the Java option `-verbose:class` (within the `tomcat/bin/catalina.sh` for the "run" command,) one can see that the `org.postgresql.Driver` class is indeed loaded, just not found at runtime, for some reason.

I can, of course, replace the `tomcat/bin/tomcat-juli.jar` with the official version, but logging then stops showing on the console. I see your `tomcat-juli.jar` `pom.xml` retrieves several packages into it, and I'll guess I'll want them for other reasons too. So, it would be nice if this worked on your end.

### PR DESCRIPTION
**Pull request by DavidGerald-Milligan**

### CODE REVIEW

File: caf-logging-tomcat-juli/pom.xml [hunk 1]
```diff
@@ -144,6 +144,7 @@
                                         <include>org/apache/juli/ClassLoaderLogManager**</include>
                                         <include>org/apache/juli/FileHandler**</include>
                                         <include>org/apache/juli/AsyncFile**</include>
+                                        <include>org/apache/juli/OneLineFormatter**</include>
```
Comment (michael-bryson) [inline caf-logging-tomcat-juli/pom.xml:147]: Why is this showing up in the PR when it was already in the merged PR: https://github.com/CAFapi/opensuse-tomcat-image/pull/42

I don't think we should be restoring already merged branches to make additional changes.

File: release-notes-5.4.0.md [hunk 2]
```diff
@@ -5,4 +5,7 @@ ${version-number}
 
 #### New Features
 
+#### Bug fixes
+ - [SCMOD-13334](https://portal.digitalsafe.net/browse/SCMOD-13334) [Tomcat Base Image Issue #41](https://github.com/CAFapi/opensuse-tomcat-image/issues/41): Tomcat-Juli Jar Causes java.lang.NoClassDefFoundError: Could not initialize class org.postgresql.Driver
```
Comment (michael-bryson) [inline release-notes-5.4.0.md:9]: This is an open source repo so no internal JIRA links should be in the release notes.

Comment (DavidGerald-Milligan) [reply release-notes-5.4.0.md:9]: I've removed the link

File: caf-logging-tomcat-juli/pom.xml (hunk 1 above)
Comment (DavidGerald-Milligan) [reply caf-logging-tomcat-juli/pom.xml:147]: It does not affect the history, we only have two commits, the original PR and the update to the release notes.

**Pull request merged**